EigenExplorer/src/handlers/strategyManager/StrategyAddedToDepositWhitelist.ts

```diff
@@ -0,0 +1,82 @@
+import { IEventContext, IBind, Instance, ISecrets } from "@blockflow-labs/utils";
+import { Strategy, Stats } from "../../types/schema";
+import { strategyAbi } from "../../data/abi/strategy";
+import { erc20Abi } from "../../data/abi/erc20";
+import { ethers } from "ethers";
+import { updateStats } from "../../utils/helpers";
+
+/**
+ * @dev Event::StrategyAddedToDepositWhitelist(address strategy)
+ * @param context trigger object with contains {event: {strategy }, transaction, block, log}
+ * @param bind init function for database wrapper methods
+ */
+export const StrategyAddedToDepositWhitelistHandler = async (
+  context: IEventContext,
+  bind: IBind,
+  secrets: ISecrets,
+) => {
+  // Implement your event handler logic for StrategyAddedToDepositWhitelist here
+
+  const { event, transaction, block, log } = context;
+  const { strategy } = event;
+
+  const rpcEndpoint = secrets["RPC_ENDPOINT"];
+  const provider = new ethers.providers.JsonRpcProvider(rpcEndpoint);
+
+  const strategyContract = new ethers.Contract(strategy, strategyAbi, provider);
+  const underlyingTokenAddress = await strategyContract.underlyingToken();
+
+  // fetch name, symbol and decimals of the underlying token
+  const underlyingTokenContract = new ethers.Contract(underlyingTokenAddress, erc20Abi, provider);
+
+  const [name, symbol, decimals] = await Promise.all([
+    underlyingTokenContract.name(),
+    underlyingTokenContract.symbol(),
+    underlyingTokenContract.decimals(),
+  ]);
```

Comment:
I think these calls can be reduced if we can use multicall. 

https://www.npmjs.com/package/ethers-multicall?activeTab=dependencies